generators/server/templates/src/main/docker/config/realm-config/jhipster-realm.json.ejs

```diff
@@ -69,6 +72,27 @@
         "containerId": "jhipster",
         "attributes": {}
       },
+      {
+        "id": "d2b73e7b-a2d7-40e9-8ebc-2af00454e8aa",
+        "name": "default-roles-jhipster",
```

Comment:
Maybe a more descriptive role?
'service_account_role'?
Also, I don't see why uma is activated.

Reply:
`default-roles-jhipster` is an internal name (auto-generated) assigned to default roles in the `jhipster` realm. 
I haven't specifically enabled `uma_authorization` and it seems another default.

Unless there is a strong reason to change, I will prefer to not update default settings.